restapi/src/main/java/io/stargate/web/resources/RowResource.java

```diff
@@ -409,11 +409,14 @@ private static Object filterToValue(Object val, String column, Table tableData)
     List<String> values = idFromPath(path);
 
     final List<Column> keys = tableMetadata.primaryKeyColumns();
-    if (keys.size() < values.size()) {
+    boolean notAllPartitionKeys = values.size() < tableMetadata.partitionKeyColumns().size();
+    boolean tooManyValues = values.size() > keys.size();
```

Comment:
would it be possible to extract this common validation logic to separate method and reuse here and:
https://github.com/stargate/stargate/pull/166/files#diff-33b3affad0379577606afe9149a9f6a3R366
?

Reply:
yes, I was thinking about it as well.
But @dougwettlaufer mentioned he will do some refactoring in the future so I decided not not mess.

I think this should be part of shared "validation" logic that does not exist yet.

Reply:
https://github.com/stargate/stargate/issues/101